starknet_py/net/signer/test_stark_curve_signer.py

```diff
@@ -1,9 +1,17 @@
+import os
+from pathlib import Path
+
 import pytest
 
+from starknet_py.common import create_compiled_contract
 from starknet_py.net.models import StarknetChainId
-from starknet_py.net.models.transaction import InvokeFunction, DeployAccount
+from starknet_py.net.models.transaction import InvokeFunction, DeployAccount, Declare
 from starknet_py.net.signer.stark_curve_signer import StarkCurveSigner, KeyPair
 
+contract_source = (
+    Path(os.path.dirname(__file__)) / "../../tests/e2e/mock/contracts/erc20.cairo"
```

Comment:
There already exists `contracts_dir` in e2e.conftest, so you could use:

```
contract_source = (
    contracts_dir / "erc20.cairo"
).read_text("utf-8")
```

Reply:
Yes, I know, but I'm not sure if we should import things from `e2e/conftest` here. I would suggest creating the main `conftest` file someday

Reply:
#441